test-support/helpers/ember-test-selectors.js

```diff
@@ -1,8 +1,6 @@
-import Ember from 'ember';
+import { deprecate } from '@ember/application/deprecations';
```

Comment:
I'm not sure we can do that in the `test-support` folder, as that one is transpiled by the host `ember-cli-babel` version, and we can't be sure that that is using `ember-cli-babel@^6.6`.